tests/acceptance/features/apiGraph/deleteUser.feature

```diff
@@ -0,0 +1,73 @@
+@api @skipOnOcV10
+Feature: delete user
+  Only user with admin permissions can delete user
+
+  Note - this feature is run in CI with ACCOUNTS_HASH_DIFFICULTY set to the default for production
+  See https://github.com/owncloud/ocis/issues/1542 and https://github.com/owncloud/ocis/pull/839
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+
+  Scenario Outline: the admin user deletes a user
+    Given the administrator has given "Alice" the role "Admin" using the settings api
```

Comment:
This can go to the Background

Reply:
We have one scenario where `Admin` role is not given to `Alice`.
`And the administrator has given "Alice" the role "<role>" using the settings api`
So that we cannot add this in the Background.